samples/build-netlify/README.md

```diff
@@ -0,0 +1,25 @@
+# Build Netlify
+
+This Sidebar Extension shows how to use Netlify API so you could trigger Netlify builds manually while editing content.
```

Comment:
maybe add something like:

This is an alternative approach to using a webhook to trigger a build. The advantage is that the editor gets control on when a build is executed instead of responding to automatic events such as `Entry.publish`.

Reply:
Added.